styles.css

```diff
@@ -0,0 +1,1985 @@
+    width: 24px;
+}
+
+input[type="radio"]:focus,
+input[type="checkbox"]:focus {
+    border: 3px solid #6bf99c;
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    background-color: #1E2019;
+    border: 3px solid #6bf99c;
+}
+
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+input[type="radio"]:checked+label,
+input[type="checkbox"]:checked+label {
+    border-bottom: 4px solid #6bf99c;
+    outline: none;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: #6bf99c;
+}
+
+input[type="text"]:required:valid,
+input[type="text"]:required:valid,
+input[type="email"]:required:valid {
+    background-color: #6bf99c;
+}
+
+input[type="text"]:default,
+input[type="email"]:default,
+input[type="text"]:default {
+    background-color: #d1d5de;
+}
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="text"]:focus {
+    background-color: #fff;
+}
+
+/*
+input[type="text"]:invalid,
+input[type="email"]:invalid,
+input[type="text"]:invalid {
+    background-color: #C61A3A;
+}
+*/
+
+
+input[type="text"]:invalid {}
+
+input[type="text"]:placeholder-shown {}
+
+input[type="text"]:not(placeholder-shown) {}
+
+input[type="text"]:not (:placeholder-shown):invalid {
+    background-color: orange;
+}
+
+textarea:focus {
+    border: 3px solid #6bf99c;
+    outline: none;
+}
+
+textarea:default {
+    background-color: #d1d5de;
+}
+
+div {
+    background-color: #1e2019;
+    color: #eaecf0;
+    min-width: auto;
+    width: 327px;
+    height: auto;
+    margin: 24px;
+    padding-bottom: 80px;
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-top: 15px;
+    /*position: relative;*/
+    /*right: 10px;*/
+}
+
+label {
+    font-size: 22px;
+    letter-spacing: 1px;
+    display: block;
+}
+
+input {
+    height: 50px;
+    width: 291px;
+    margin-bottom: 15px;
+    margin-top: 12px;
+    margin-right: 0px;
+}
+
+input#age {
+    width: 75px;
+    display: block;
+    margin-top: 5px;
+}
+
+textarea#selfdescribe {
+    margin-bottom: 40px;
+    width: 291px;
+}
+
+textarea#powers {
+    height: 175px;
+    width: 291px;
+}
+
+
+legend {
+    font-family: 'BenchNine', sans-serif;
+    font-size: 22px;
+    font-weight: 700;
+}
+
+p {
+    font-size: 20px;
+    font-weight: 400;
+    color: #D1D5DE;
+    line-height: 22px;
+    margin-bottom: 5px;
+    margin-top: 5px;
+}
+
+.abilities {
+    border-bottom: 4px solid #163346;
+    color: #eaecf0;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    font-weight: 400px;
+    height: 35px;
+    letter-spacing: 1px;
+    margin-bottom: 30px;
+    width: 235px;
+}
+
+.rate {
+    color: #eaecf0;
+    font-size: 22px;
+    margin-bottom: 10px;
+    margin-top: 10px;
+}
+
+.svg {
+    background-color: transparent;
+    display: inline-block;
+    left: 60px;
+    margin: 0;
+    margin-left: 75px;
+    padding: 0;
+    width: 70%;
+    /*right: 45px;*/
+    /*padding-bottom: 40px;*/
+}
+
+.strength {
```

Comment:
I would look for the similarities between your `strength`, `speed` and `intelligence` classes -- where possible, you want to reuse code and not have to duplicate it and then maintain it. Since these are almost exactly the same component (`svg`+`label`+`select`), it should be possible to write CSS that applies to all three and give them a shared classname.